py/server/tests/test_numba_vectorized_column.py

```diff
@@ -32,6 +35,22 @@ def test_column(self):
         html_output = to_html(t)
         self.assertIn("<td>9</td>", html_output)
 
+    def test_boxed_type_arg(self):
+        @vectorize(['float64(float64)'])
+        def norm_cdf(x):
+            """ Cumulative distribution function for the standard normal distribution """
+            return (1.0 + math.erf(x / math.sqrt(2.0))) / 2.0
+
+        # make sure we don't get a warning about numpy scalar used in annotation
+        warnings.filterwarnings("error", category=UserWarning)
```

Comment:
How is this asserting that there is no warning?

Reply:
If there is a warning we would get an Error that would fail the test.